composer.json

```diff
@@ -19,7 +19,7 @@
         "ext-json": "*",
         "phpdocumentor/reflection-docblock": "^4.4|^5.0",
         "ruflin/elastica": "^7.0",
-        "symfony/deprecation-contracts": "^2.4",
+        "symfony/deprecation-contracts": "^2.4|^3.0",
```

Comment:
could you use the `||` notation, like above ? with only one `|` it's deprecated

Reply:
Of course, I update it